Backend/UnhousedOutreach.Api/Controllers/LookupsController.cs

```diff
@@ -58,6 +58,19 @@ public async Task SetDisability([FromBody]KeyValuePair<int?, string> disability,
         await repository.SetDisability(disability.Key, disability.Value, outreachTeamId);
     }
 
+
+    [HttpGet("english-levels")]
+    public async Task<Dictionary<int, string>> GetEnglishLevelLookup([FromQuery(Name = "otid")]int outreachTeamId)
```

Comment:
Could this method be renamed to the plural? `GetEnglishLevelsLookup`

Reply:
Fixed.